install/mysql/glpi-9.2-empty.sql

```diff
@@ -7040,6 +7040,7 @@ CREATE TABLE `glpi_taskcategories` (
   `entities_id` int(11) NOT NULL DEFAULT '0',
   `is_recursive` tinyint(1) NOT NULL DEFAULT '0',
   `taskcategories_id` int(11) NOT NULL DEFAULT '0',
+  `knowbaseitemcategories_id` int(11) NOT NULL DEFAULT '0',
```

Comment:
OK sounds like this permit to do something that was not possible before.. Ok for me.